app/api/resources/metadata.py

```diff
@@ -259,3 +260,30 @@ async def get_genome_by_keyword(request: Request, assembly_accession_id: str):
     except Exception as ex:
         logging.error(ex)
         return response_error_handler({"status": 500})
+
+@router.get("/genome/{genome_uuid}/vep/file_paths")
```

Comment:
in line 210 of this file we have implementation of dataset attributes which is used in toolsAPI: https://github.com/Ensembl/ensembl-web-tools-api/blob/main/app/vep/utils/web_metadata.py#L15
What is the difference between these two?

Reply:
Good question, as far as I understood, Daniel from Automation wanted to avoid using `attribute name` to get the file path, like what we are doing in line 210 ([example](https://beta.ensembl.org/api/metadata/genome/a73357ab-93e7-11ec-a39d-005056b38ce3/dataset/genebuild/attributes?attribute_names=vep.faa_location&attribute_names=vep.gff_location)).

The requirement was: _For a given `genome_uuid` get me the VEP file paths_ (without going through the attribute name)

I hope this makes sense.

> I also updated toolsAPI in this PR https://github.com/Ensembl/ensembl-web-tools-api/pull/55 to reflect these changes.

Reply:
Great!
 Thanks Bilal!